src/consts/strings.js

```diff
@@ -0,0 +1,3 @@
+export const STRING_METADATA_SEARCH_HINT_TITLE = ' Suggestion ';
+export const STRING_METADATA_INPUT_PLACEHOLDER = 'Metadata Search';
```

Comment:
I don't think exporting `STRING_METADATA_INPUT_PLACEHOLDER` and `STRING_SEARCH_BUTTON` is needed since it is only used `src/react/databrowser/objects/MetadataSearch.jsx`. What do you think?

Reply:
My idea with string.js was, starting to store static strings if you want to handle multilang later, but I can revert string.js and just keep consts/index for now if you want